task/clair-scan/0.1/README.md

```diff
@@ -0,0 +1,30 @@
+# clair-scan task
+
+## Description:
+The clair-scan task performs vulnerability scanning uses Clair, an open source tool for performing static analysis
```

Comment:
```suggestion
The clair-scan task performs vulnerability scanning using Clair, an open source tool for performing static analysis
```

Reply:
Thanks for the good comments @hongweiliu17, you caught numerous typos I glanced right over. They should all be fixed in the most recent patch.